src/js/components/List/stories/PaginatedList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import { Box, Grommet, List, Menu } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { More } from 'grommet-icons';
+
+const data = [];
+
+for (let i = 0; i < 95; i += 1) {
+  data.push({
+    entry: `entry-${i + 1}`,
+  });
+}
+
+export const PaginatedList = () => (
+  <Grommet theme={grommet} full>
+    <Box pad="medium">
+      <List
+        data={data}
+        action={(item, index) => {
+          return (
+            <Menu
+              key={index}
+              icon={<More />}
+              hoverIndicator
+              items={[{ label: 'one' }]}
+            />
+          );
+        }}
+        step={3}
+        show={{ page: 7 }}
+        paginate={{ margin: { top: 'medium' } }}
```

Comment:
Good callout to remove custom margin. It revealed that this would be the default List behavior. It feels like we should put some space between the List and pagination by default, but that the user could override this if desired. What do you think?

<img width="1459" alt="Screen Shot 2021-01-20 at 4 32 39 PM" src="https://user-images.githubusercontent.com/12522275/105258430-9d4b7480-5b3e-11eb-8f1d-e7efc16250b5.png">

Reply:
The default case should look reasonable. This sounds like some default margin in the base theme might be in order.